.buildkite/scripts/release-manager.sh

```diff
@@ -20,9 +20,6 @@ set -uexo pipefail
 readonly TYPE=${TYPE:-snapshot}
 readonly OUTPUT_FILE=${OUTPUT_FILE:-release-manager-report.out}
 
-# shellcheck source=/dev/null
-source /usr/local/bin/bash_standard_lib.sh
```

Comment:
What was the needed function from this file ?

Reply:
Why are seeing that?
It was removed days ago...